src/main/resources/migration/V1_2__credit_advisory.sql

```diff
@@ -0,0 +1,78 @@
+CREATE SCHEMA IF NOT EXISTS credit_advisory;
+
+DO $$ BEGIN
+CREATE TYPE credit_advisory.advisors_role AS ENUM('associate', 'partner', 'senior');
+        EXCEPTION WHEN duplicate_object THEN NULL;
+END $$;
+
+CREATE TABLE IF NOT EXISTS credit_advisory.advisors (
+    id SERIAL CONSTRAINT PK_advisors PRIMARY KEY,
+    email VARCHAR(255) NOT NULL CONSTRAINT UQ_advisors_email UNIQUE,
+    username VARCHAR(255) NOT NULL,
+    role advisors_role NOT NULL,
+    created_at TIMESTAMP DEFAULT NOW() NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS credit_advisory.applicants (
+    id SERIAL CONSTRAINT PK_applicants PRIMARY KEY,
+    email VARCHAR(255) NOT NULL CONSTRAINT UQ_applicants_email UNIQUE,
+    username VARCHAR(255) NOT NULL,
+    first_name VARCHAR(255) NOT NULL,
+    last_name VARCHAR(255) NOT NULL,
+    social_security_number VARCHAR(255) NOT NULL,
+    created_at TIMESTAMP DEFAULT NOW() NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS credit_advisory.addresses (
+    applicant_id INT CONSTRAINT PK_addresses PRIMARY KEY,
+    city VARCHAR(255),
+    street VARCHAR(255),
+    number VARCHAR(10),
+    zip VARCHAR(10),
```

Comment:
I believe these columns should not be `NULL` as it's important to know applicant's address for the advisor.